clustered/client/src/test/java/org/ehcache/clustered/docs/GettingStarted.java

```diff
@@ -47,7 +49,7 @@ public void clusteredCacheManagerExample() throws Exception {
     // tag::clusteredCacheManagerExample
     final CacheManagerBuilder<PersistentCacheManager> clusteredCacheManagerBuilder =
         CacheManagerBuilder.newCacheManagerBuilder()
-            .with(new ClusteringServiceConfiguration(URI.create("http://example.com:9540/my-application?auto-create")))
+            .with(ClusteringServiceConfigurationBuilder.cluster(URI.create("http://example.com:9540/my-application?auto-create")).build())
```

Comment:
`.build()` is optional here normally and thus better removed from this doc sample